deploy/blobfuse2/config.yaml

```diff
@@ -0,0 +1,23 @@
+allow-other: true
+
+logging:
+  type: base
+  level: log_debug
```

Comment:
Do we want to keep log_debug as default option, it will dump good amount of logs and overall performance will also hit. What do we keep as default logging for blobfuse ? if nothing is configured then its default to log_warning.